README.md

```diff
@@ -218,19 +218,45 @@ When running OpenShift in a single VM, it is recommended to reuse the Docker dae
 for pure Docker use-cases as well.
 By using the same docker daemon as Minishift, you can speed up your local experiments.
 
-To be able to work with the docker daemon on your Mac or GNU/Linux host use the
-[`docker-env command`](https://minishift.io/docs/minishift_docker-env.html) in your shell:
+1. Make sure that you have `docker` and `docker-client` installed on your machine.
```

Comment:
" Make sure that you have `docker` and `docker-client` installed on your machine."

"docker" and "docker-client" are rpm package names. They don't exist on other OS types and they are also not consistently names across Linux distributions. What we can write is that you need the "Docker client binary" installed on your system. How to do this varies a lot. Check out https://docs.docker.com/engine/installation/ to see how varied it can be.